build.gradle

```diff
@@ -255,6 +256,21 @@ subprojects {
       }
     }
 
+    slf4jApi = { ->
+     dependencies {
+        compileOnly group: 'org.slf4j', name: 'slf4j-api', version: slf4jVersion
+      }
+    }
+
+    slf4jLambdaApi = { ->
+      dependencies {
+        slf4jApi()
+        compile(group: 'kr.pe.kwonnam.slf4j-lambda', name: 'slf4j-lambda-core', version: slf4jLambdaApiVersion)  {
+          exclude(group: 'org.slf4j')
```

Comment:
Nit: just exclude `slf4j-api` explicitly? Or do we need to exclude the entire group?

Reply:
I think that excluding entire group is safer.